libs/components/ag-grid/src/lib/modules/ag-grid/cell-validator/ag-grid-cell-validator-tooltip.component.ts

```diff
@@ -50,6 +49,14 @@ export class SkyAgGridCellValidatorTooltipComponent {
       }
     });
 
+    this.cellRendererParams.eGridCell?.addEventListener('mouseover', () => {
```

Comment:
The `mouseover` and `mouseout` events can be expensive events to listen to. Can the feature be resolved with `mouseenter` and `mouseleave`?

Reply:
Switched the events.